WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -38,31 +38,32 @@
           + "title,"                // 9
           + "text,"                 // 10
           + "excerpt,"              // 11
-          + "url,"                  // 12
-          + "short_url,"            // 13
-          + "blog_url,"             // 14
-          + "blog_name,"            // 15
-          + "featured_image,"       // 16
-          + "featured_video,"       // 17
-          + "post_avatar,"          // 18
-          + "sort_index,"           // 19 - this is a score for search results, otherwise it's a timestamp
-          + "published,"            // 20
-          + "num_replies,"          // 21
-          + "num_likes,"            // 22
-          + "is_liked,"             // 23
-          + "is_followed,"          // 24
-          + "is_comments_open,"     // 25
-          + "is_external,"          // 26
-          + "is_private,"           // 27
-          + "is_videopress,"        // 28
-          + "is_jetpack,"           // 29
-          + "primary_tag,"          // 30
-          + "secondary_tag,"        // 31
-          + "attachments_json,"     // 32
-          + "discover_json,"        // 33
-          + "word_count,"           // 34
-          + "xpost_post_id,"        // 35
-          + "xpost_blog_id";        // 36
+          + "format,"               // 12
```

Comment:
I wonder why you decided to insert the new column here, and then needing to move all the rest of the parameters down 😱 
But it looks better in code though